src/pages/contact.js

```diff
@@ -1,4 +1,4 @@
-import './contact.css'
+import './contact.css';
```

Comment:
This isn't C++ 🤣. Could you remove the semicolon for consistency lol

Reply:
Semicolons are an essential part of JavaScript code. They are read and used by the compiler to distinguish between separate statements so that statements do not leak into other parts of the code

Reply:
I understand that, I noticed that we are using them improperly in other parts of the project, so we will go back and remove them at some point. Here the semicolon has no use though.